src/components/Sale/SelectedResource.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-container v-if="props.resources.value?.length > 0">
+    <div class="mx-auto text-center" style="font-size: 16px">
+      Currently selected resources:
+    </div>
+    <ResourcePriceDiscountRow
+      v-for="(resource, index) in props.resources.value"
+      :key="index"
+      :resource="resource"
+    >
+      <SaleToolTipResource :sale-data="props.allResources" :resource="resource" />
+    </ResourcePriceDiscountRow>
+  </v-container>
+
+  <v-container
+    v-if="props.resources.value?.length > 0"
+    class="d-flex flex-column mt-4"
+  >
+    <h5>Resources:</h5>
+    <p>Price: € {{ currentResourcePrice.toFixed(2) }}</p>
+    <p>Discount: {{ computedResourcePrice.toFixed(2) }} %</p>
+    <p>Final price: € {{ finalResourcePrice.toFixed(2) }}</p>
+  </v-container>
```

Comment:
This looks like very similar to the container in the SelectedProduct. Perhaps it can become a component?

Reply:
i would leave it like that, better for reading and understanding